framework/src/modules/http_api/controllers/accounts.js

```diff
@@ -84,6 +84,12 @@ function accountFormatter(totalSupply, account) {
  * @todo Add description for the function and the params
  */
 AccountsController.getAccounts = async function(context, next) {
+	const invalidParams = swaggerHelper.invalidParams(context.request);
```

Comment:
Could you increase the functional test coverage of every controller where you are addressing this?

Reply:
@diego-G Added a lot of tests!